tfutils/base.py

```diff
@@ -119,6 +119,8 @@ def __init__(self,
         self.cache_filters_freq = cache_filters_freq
         self.save_filters_freq = save_filters_freq
         self.save_initial = save_initial
+        self._restore = restore
```

Comment:
added "_restore" property for use later in load_model.   it's not called "restore" because that's a method of the base class we want to not override.